managed/ui/src/components/tables/ListBackups/ListBackups.js

```diff
@@ -283,6 +285,42 @@ export default class ListBackups extends Component {
     });
   };
 
+  /**
+   * This method will help us to get the particular backupUUID
+   * for the selected row.
+   *
+   * @param {string} {backupUUID} Backup UUID.
+   * @param {boolean} isSelected Selected row.
+   * @returns Boolean
+   */
+  onRowSelect = ({ backupUUID }, isSelected) => {
+    isSelected ?
+      this.setState({ selected: [...this.state.selected, backupUUID].sort() }) :
+      this.setState({ selected: this.state.selected.filter((id) => id !== backupUUID) });
+
+    return false;
+  };
+
+  /**
+   * This method will help us to select all the backups for
+   * the current page.
+   *
+   * @param {boolean} isSelected Selected rows.
+   * @param {Array} rows Number of rows.
+   * @returns Boolean
+   */
+  onSelectAll = (isSelected, rows) => {
+    let selected = [...this.state.selected];
+    if (isSelected) {
+      rows.forEach((row) => selected.push(row.backupUUID));
+      this.setState({ selected: selected });
```

Comment:
Just move the variable declaration into the if-block: `const selected = [...this.state.selected, ...rows.map(row => row.backupUUID)]`

Reply:
Done